src/components/Sum.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = ({
+  numbers: { x, y },
+}) => (
+  <p>
+    {`Sum of ${x} and ${y} id ${x + y}`}
+  </p>
```

Comment:
```suggestion
  <p>
    {
      `Sum of ${x} and ${y} is ${x + y}`
    }
  </p>
```
Немного отфарматируй и опечатку на **is** поменяй